exporter/annotations.go

```diff
@@ -0,0 +1,114 @@
+package exporter
+
+import (
+	"fmt"
+
+	ocispecs "github.com/opencontainers/image-spec/specs-go/v1"
+
+	"github.com/containerd/containerd/platforms"
+	"github.com/moby/buildkit/exporter/containerimage/exptypes"
+)
+
+type Annotations struct {
```

Comment:
I wonder if this should be either in `exptypes` or in `exporter/containerimage`. It is only image-specific afaics.